subt/script/run_skiddy.bash

```diff
@@ -3,5 +3,8 @@
 source ros/subt/setup.bash
 export PYTHONPATH=${PWD}:${PYTHONPATH}
 export OMP_NUM_THREADS=1  # Make OpenCV eat fewer cpus.
+python3 -m osgar.record config/skiddy-subt-serialloop.json &
+pidcore=$!
 python3 -m subt.main --use-old-record run config/skiddy-subt.json --side left --speed 0.4 --timeout 180 --wall-dist 1.0 --gap-size 1.0
 #python3 -m osgar.go run config/skiddy-subt.json -s 0.3 -d -4 --timeout 30
+kill "$pidcore"
```

Comment:
Is the serialloop correctly terminated when you CTRL+C while subt.main is running?

Reply:
I think yes, but I am not 100% sure as I was receiving resource errors from ROS and could not kill it just with Ctrl+C. I will check it again, thanks

Reply:
It seems to be working OK (there was just small typo in JSON " vs. ')